persistence/scaladsl/src/main/scala/com/lightbend/lagom/internal/scaladsl/persistence/ReadSideActor.scala

```diff
@@ -78,6 +89,7 @@ private[lagom] class ReadSideActor[Event <: AggregateEvent[Event]](
   def receive = {
     case EnsureActive(tagName) =>
       implicit val timeout = Timeout(config.globalPrepareTimeout)
+      projectorRegistryActorRef ! ProjectorRegistryActor.RegisterProjector(streamName, projectorName, tagName)
```

Comment:
We only need to register once. We can track it with some state inside that actor and only do it on the first `EnsureActive`

Reply:
Hmmm, no. A few lines after this `tell()` the actor `become()`s `active()` and that behavior ignores `EnsureActive`. ;-)